packages/functions/src/cache/in-memory-cache.ts

```diff
@@ -0,0 +1,67 @@
+import { FunctionCache } from './types';
+
+interface CacheEntry {
+  value: unknown;
+  lastModified: number;
```

Comment:
Ditto: Is this a UNIX time in milliseconds?

Reply:
updated